utils/zerovec/derive/examples/make.rs

```diff
@@ -0,0 +1,84 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use std::fmt::Debug;
+use zerovec::*;
+use zerovec_derive::*;
+
+#[make_ule]
+#[derive(Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord)]
+struct Struct {
```

Comment:
it becomes really annoying to do the to/from impls, and I think one of the main reasons I want this is so that you can make ULE types without having to think about it: I've already seen people find this somewhat challenging at times, especially around VarULE. We can document the macro output better (I plan to do a doc pass once all the features are in)

I'd rather have the automagic struct generation.

Reply:
Besides, as we add structpacking features we'll _really_ need this.

Reply:
OK.